### PR TITLE
Removed unnecessary periods and fixed typos in Imprint

### PR DESCRIPTION
**Pull request by g9militantsYT**
Btw its my first time reading an imprint so subconsciously I think two possibilities are there: 1. Lawyers are poor in english 2. Imprint was rushed.
Please review the 2nd commit (I really don't wanna be sued 🙏)
In the first commit I removed some periods (unnecessary) that were present in the headings and such. 
I didnt wanna touch the Imprint a lot (scared to death) so you might wanna review if sentences in the Imprint should end with a period (they don't, yet) and in some places sentences start after a period rightaway with the absence of a whitespace (eg: ...s are protected by copyright.Their download, reproduction or use...)

**TLDR by GPT: Reviewing the second commit to ensure it's legally sound. Initially, removed unnecessary periods but didn't heavily edit the Imprint section. Concerned about potential legal issues due to lack of proper grammar and rushed publication.**

> As a sidenote, I need to do some actual stuff instead of reading and fixing legal pages (who actually reads this 💀 )

### CODE REVIEW

**Comment by lucsoft** [comment]
Why do you always want to remove these periods? Just because its technically not necessary?

**Comment by g9militantsYT** [comment]
> Why do you always want to remove these periods? Just because its technically not necessary?

Periods in headings look gross tbh. See the below site for eg:

image

Since when are periods even used in headings bruh

File: pages/holding/imprint.ts [hunk 1]
```diff
@@ -22,16 +22,16 @@ Body(Box(
         ).addClass("block").setAlignItems("center"),
         Label(`Contents`, "h3"),
         Label(
-            `BBN Holding regularly examines and updates the contents of this website. However we give no guarantee for the accuracy, completeness, timeliness or reliability of the information provided and are not liable for claims resulting directly or indirectly from flawed or erroneous informat`,
+            `BBN Holding regularly examines and updates the contents of this website. However we give no guarantee for the accuracy, completeness, timeliness or reliability of the information provided and are not liable for claims resulting directly or indirectly from flawed or erroneous information`,
```
Comment (lucsoft) [inline pages/holding/imprint.ts:25]:
```suggestion
            `BBN Holding regularly examines and updates the contents of this website. However we give no guarantee for the accuracy, completeness, timeliness or reliability of the information provided and are not liable for claims resulting directly or indirectly from flawed or erroneous information.`,
```

File: pages/holding/imprint.ts [hunk 2]
```diff
@@ -22,16 +22,16 @@ Body(Box(
...
-            `In order to offer our customers additional information we occasionally provide links to other websites. We herewith state explicitly that we have no influence on the set-up, the timeliness or the contents of linked pages. For this reason we distance ourselves expressly from all contents on the pages linked with this Homepage and disclaim all forms of liability for these as well. We are not aware of any content which is illegal, objectionable or contrary to the rules of fair competition. Please inform BBN Holding if you believe that a linked page infringes upon these princip`,
+            `In order to offer our customers additional information we occasionally provide links to other websites. We herewith state explicitly that we have no influence on the set-up, the timeliness or the contents of linked pages. For this reason we distance ourselves expressly from all contents on the pages linked with this Homepage and disclaim all forms of liability for these as well. We are not aware of any content which is illegal, objectionable or contrary to the rules of fair competition. Please inform BBN Holding if you believe that a linked page infringes upon these principles`,
```
Comment (lucsoft) [inline pages/holding/imprint.ts:28]:
```suggestion
            `In order to offer our customers additional information we occasionally provide links to other websites. We herewith state explicitly that we have no influence on the set-up, the timeliness or the contents of linked pages. For this reason we distance ourselves expressly from all contents on the pages linked with this Homepage and disclaim all forms of liability for these as well. We are not aware of any content which is illegal, objectionable or contrary to the rules of fair competition. Please inform BBN Holding if you believe that a linked page infringes upon these principles.`,
```

File: pages/holding/imprint.ts (hunk 1 above)
Comment (g9militantsYT) [reply pages/holding/imprint.ts:25]: https://www.thesaurus.com/e/grammar/period/

File: pages/holding/imprint.ts [hunk 3]
```diff
@@ -22,16 +22,16 @@ Body(Box(
...
         ).addClass("block"),
         Label(`Netiquette and Site Guidelines`, "h3"),
         Label(
-            `For many years, BBN Holding strives to provide helpful and relevant information on the BBN Holding company website. BBN Holding also shares news and information about latest events on maritime\xA0websites or social platforms like LinkedIn. We are happy to connect with you and read your feedb`,
+            `For many years, BBN Holding strives to provide helpful and relevant information on the BBN Holding company website. BBN Holding also shares news and information about latest events on maritime\xA0websites or social platforms like LinkedIn. We are happy to connect with you and read your feedback`,
```
Comment (lucsoft) [inline pages/holding/imprint.ts:32]:
```suggestion
            `For many years, BBN Holding strives to provide helpful and relevant information on the BBN Holding company website. BBN Holding also shares news and information about latest events on maritime\xA0websites or social platforms like LinkedIn. We are happy to connect with you and read your feedback!`,
```

File: pages/holding/imprint.ts [hunk 4]
```diff
@@ -22,16 +22,16 @@ Body(Box(
...
         ).addClass("block"),
-        Label(`Please keep in mind: comments should be relevant and on-topic. Inappropriate or offensive comments, and in particular those that contain false\xA0/ misleading information or that engage in personal attacks (and similar), may be dele`).addClass("block"),
+        Label(`Please keep in mind: comments should be relevant and on-topic. Inappropriate or offensive comments, and in particular those that contain false\xA0/ misleading information or that engage in personal attacks (and similar), may be deleted`).addClass("block"),
```
Comment (lucsoft) [inline pages/holding/imprint.ts:34]:
```suggestion
        Label(`Please keep in mind: comments should be relevant and on-topic. Inappropriate or offensive comments, and in particular those that contain false\xA0/ misleading information or that engage in personal attacks (and similar), may be deleted.`).addClass("block"),
```

File: pages/holding/imprint.ts (hunk 1 above)
Comment (g9militantsYT) [reply pages/holding/imprint.ts:25]: Doesn't hurt to have a good looking Imprint, just saying

**Review by lucsoft — CHANGES_REQUESTED** [review]

**Comment by g9militantsYT** [comment]
image
Pull?

**Comment by lucsoft** [comment]
you mean Merge

**Comment by lucsoft** [comment]
can you update the last label block from "trademark" to "trademarks."

**Comment by g9militantsYT** [comment]
> you mean Merge

yeah that lol. running 2 convos at the same time results in erranous output

**Comment by lucsoft** [comment]
missed the period 💀

**Comment by g9militantsYT** [comment]
> missed the period 💀

wait I can add the periods then? alr

**Comment by g9militantsYT** [comment]
Cries in " Harvey Specter"

image

**Comment by g9militantsYT** [comment]
Oopsies. We had a short load shedding and the inverter gave out 💀 anyway electricity is back

**Comment by g9militantsYT** [comment]
anyway, yeah i did the last period this..

**Review by lucsoft — APPROVED** [review]
LGTM :shipit:

**Comment by g9militantsYT** [comment]
uwu
